### PR TITLE
fix failing regionprops test cases

### PR DESCRIPTION
**Pull request by grlee77**
This PR should fix the regionprops test failures [seen recently on CI](https://gpuci.gpuopenanalytics.com/job/rapidsai/job/gpuci/job/cucim/job/prb/job/cucim-gpu-test/CUDA=11.0,GPU_LABEL=gpu-a100,LINUX_VER=centos7,PYTHON=3.7/51/console). Seems related to #4354, but not sure why the prior fix no longer works.

The image is always boolean, so a cast to uint8 should be a safe.

cc @gigony

### CODE REVIEW

File: python/cucim/src/cucim/skimage/measure/_regionprops.py [hunk 1]
```diff
@@ -311,9 +311,9 @@ def convex_image(self):
         # from ..morphology.convex_hull import convex_hull_image
         from skimage.morphology.convex_hull import convex_hull_image
 
-        # CuPy Backend: copy required here to avoid unexpected behavior
+        # CuPy Backend: explicitly cast to uint8 to avoid the issue see in
         #               reported in https://github.com/cupy/cupy/issues/4354
-        return cp.asarray(convex_hull_image(cp.asnumpy(self.image))).copy()
+        return cp.asarray(convex_hull_image(cp.asnumpy(self.image))).astype(cp.uint8)
```
Comment (gigony) [inline python/cucim/src/cucim/skimage/measure/_regionprops.py:316]:
```suggestion
        return cp.asarray(convex_hull_image(cp.asnumpy(self.image))).astype(
            cp.uint8)
```

**Review by gigony — APPROVED** [review]
Looks good to me but somehow there are code style issues on this line

```
./src/cucim/skimage/measure/_regionprops.py:316:81: E501 line too long (85 > 80 characters)
```

There are other issues with isort but ignored by CI/CD.
https://gpuci.gpuopenanalytics.com/blue/organizations/jenkins/rapidsai%2Fgpuci%2Fcucim%2Fprb%2Fcucim-style/detail/cucim-style/65/pipeline

File: python/cucim/src/cucim/skimage/measure/_regionprops.py (hunk 1 above)
Comment (grlee77) [reply python/cucim/src/cucim/skimage/measure/_regionprops.py]: thanks!

**Comment by quasiben** [comment]
Thanks @grlee77

**Review by quasiben — APPROVED** [review]

**Pull request merged**